task1/Program.cs

```diff
@@ -45,5 +62,452 @@ public static decimal PerformCalculation(int r)
             decimal square = Pi * r * r;
             return square;
         }
+
+        public static void arrayAnalyze()
+        {
+            Console.WriteLine("Enter number of elements");
+            int arraySize = Convert.ToInt32(Console.ReadLine());
+
+            var random = new Random();
+            int[] array = new int[arraySize];
+            for (int i = 0; i < arraySize; i++)
+            {
+                array[i] = random.Next(100);
+                Console.WriteLine(array[i]);
+            }
+
+            int min = 100;
+            int max = 0;
+            int sum = 0;
+            double mean = 0.0;
+            string oddNumbers = "";
+            for (int i = 0; i < array.Length; i++)
+            {
+                // Max & min
```

Comment:
Do you remember we discussed how the comments should look like?
Comment should consists of 3 lines.
Each sentence of comment should ended with dot.